airflow/contrib/hooks/redshift_hook.py

```diff
@@ -20,7 +20,7 @@
 
 import warnings
 
-from airflow.providers.amazon.aws.hooks.redshift import RedshiftHook  # noqa
+from airflow.providers.amazon.aws.hooks.redshift_cluster import RedshiftHook  # noqa
 
 warnings.warn(
     "This module is deprecated. Please use `airflow.providers.amazon.aws.hooks.redshift`.",
```

Comment:
Need to also change these warning messages

Reply:
done